src/styles/custom.scss

```diff
@@ -1,14 +1,79 @@
+@import "~bootstrap/scss/bootstrap";
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,400;0,500;1,700&display=swap');
+
 $body-bg: #F8F9FA;
 $montserrat: 'Montserrat', sans-serif;
 $lora: 'Lora', serif;
 $open-sans: 'Open Sans', sans-serif;
 $main-font-color: #0C2028;
+$body-bg: #F8F9FA;
+$secondary-font-color: #2F4550;
+$italic-color: #537C8C;
 $primary: #82B29A;
 $orange: #E17B60;
 
 body {
-    font-family: $montserrat;
     color: $main-font-color;
+    font-family: $montserrat;
+    letter-spacing: 0.07em;
+    background-color: $body-bg;
+    h1 {
```

Comment:
We want to move as many colors and fonts as possible to our styleVars file in utils, then use our Titles file to define components with our different title components. This uses the CSS in JS way of defining highly focused React components with styles, rather than relying on higher level SCSS definitions.

Reply:
Moved the font updates to App.tsx